src/App.js

```diff
@@ -10,15 +10,37 @@ import { useAuthState } from 'react-firebase-hooks/auth';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 firebase.initializeApp({
-  apiKey: "AIzaSyAAFK3Q7X1mtyqNtezi0A1TnVutjShk2wY",
-  authDomain: "fir-chat-7c8f8.firebaseapp.com",
-  projectId: "fir-chat-7c8f8",
-  storageBucket: "fir-chat-7c8f8.appspot.com",
-  messagingSenderId: "468958569699",
-  appId: "1:468958569699:web:6da1177bced5e19f7891d6",
-  measurementId: "G-7FQ7R5RLNV"
+  apiKey: "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
+  authDomain: "chat-app-15946.firebaseapp.com",
+  projectId: "chat-app-15946",
+  storageBucket: "chat-app-15946.appspot.com",
+  messagingSenderId: "533178884935",
+  appId: "1:533178884935:web:ef9483d776d0107ffe6262",
+  measurementId: "G-P58VBCQ0SY"
 })
 
+function Timebomb() {
```

Comment:
Code is suspicious and it is not related to the application so, It will not be approved.